### DISCUSSION TITLE
Does Bevy have a module for plotting?

### DISCUSSION THREAD

**Discussion by IndianBoy42** [bevyengine/bevy#2902] [Q&A]
I am starting a project, that actually isn't really a game but it does involve 2d graphics/animations, interaction and realtime plots. So I was thinking bevy might be a good fit, as it is something I want to learn anyway.

**Comment by bjorn3** [comment]
You could take a look at https://bevyengine.org/assets/ for a list of third party plugins. I don't see a plot plugin, but bevy_prototype_lyon could be used for manually drawing plots I think.
